### PR TITLE
Other user/dutch and poland

### PR DESCRIPTION
**Pull request by shanth5100**

### CODE REVIEW

File: androidapp/utils/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<resources>
+    <!-- Abbreviate form of Week days -->
+    <string name="monday">Maa</string>
+    <string name="tuesday">Din</string>
+    <string name="wednesday">Woe</string>
+    <string name="thursday">Don</string>
+    <string name="friday">Vri</string>
+    <string name="saturday">Zat</string>
```
Comment (vsurwshe) [inline androidapp/utils/src/main/res/values-nl/strings.xml:8]: Test review

**Review by vsurwshe — COMMENTED** [review]
Test Review added

File: androidapp/utils/src/main/res/values-nl/strings.xml (hunk 1 above)
Comment (vj12354) [reply androidapp/utils/src/main/res/values-nl/strings.xml:8]: what is test review?

**Review by vj12354 — APPROVED** [review]
Approved.

**Pull request merged**